pkg/Microsoft.ML.MatrixFactorization/Microsoft.ML.MatrixFactorization.nupkgproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>LIBMF, the core computation library for matrix factorization in ML.NET</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup> 
+    <Content Include="..\common\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
```

Comment:
We should probably be including `libmf`'s notice in this package (assuming we have legal signoff).

You can see how we do it for the Mkl and Tensorflow Redist packages:

[https://github.com/dotnet/machinelearning/blob/master/pkg/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.nupkgproj#L12](https://github.com/dotnet/machinelearning/blob/master/pkg/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.nupkgproj#L12)
[https://github.com/dotnet/machinelearning/blob/master/pkg/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.nupkgproj#L19](https://github.com/dotnet/machinelearning/blob/master/pkg/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.nupkgproj#L19)

Reply:
I expect that I need to do something like `<Content Include=PATH_TO_LIBMF_COPYRIGHT Pack="true" PackagePath=".\" />` but I don't know what environment variables I can use to make `PATH_TO_LIBMF_COPYRIGHT`. I have googled the extension `nupkgproj` but got no document.

Reply:
Thank you. By running `build --buildPackages` on my machine, I can find COPYRIGHT in `D:\Repos\machinelearning\bin\packages\Microsoft.ML.MatrixFactorization.0.7.0-preview-27023-0.nupkg` now.